docs/guides/end-to-end-testing/migration/selenium-to-cypress.mdx

```diff
@@ -0,0 +1,398 @@
+requests or responses is entirely feasible using Cypress's native intercept functionality. This plugin enhances both
+the stability and usability of the intercept feature: [Shopify/cypress-graphql](https://github.com/Shopify/cypress-graphql)
+
+## Migrating test cases with Cucumber
+
+If you're currently using a Cucumber framework, [this Cypress plugin](https://github.com/badeball/cypress-cucumber-preprocessor) can assist you
+in transitioning smoothly. Although it alters the conventional `describe` and `it` functionality, the adjustment process should
+be relatively straightforward.
+
+The main difference would be that instead of adding the whole flow into a single `it` section, you add the different steps into
+an `it` like section, where the `it` is replaced with the standard Gherkin syntax.
+
+The first snippet comprises plain Cypress code, whereas the second snippet demonstrates the Cucumber version.
+
+```javascript
+it("I should see a search bar", () => {
+  cy.visit("https://www.duckduckgo.com");
+  cy.get("input").should(
+    "have.attr",
+    "placeholder",
+    "Search the web without being tracked"
+  )
+```
+
+```javascript
+import { When, Then } from '@badeball/cypress-cucumber-preprocessor'
+
+When('I visit duckduckgo.com', () => {
+  cy.visit('https://www.duckduckgo.com')
+})
+
+Then('I should see a search bar', () => {
+  cy.get('input').should(
+    'have.attr',
+    'placeholder',
+    'Search the web without being tracked'
+  )
+})
+```
+
+<br />
+As evident, the actual code or logic remains unchanged. The difference lies in the
+structure of the spec file and the framework's architecture.
+
+## Integration with CI/CD Pipeline
+
+### Integration with the current Pipeline
+
+Cypress can be integrated into any pipeline technology using its headless runner. This headless runner leverages
+the Electron browser, ensuring minimal adaptation of the current CI/CD. Given the widespread usage of Docker in
+most pipelines today, the Cypress pipeline is no exception. Integration becomes effortless by employing a Node.js
+docker image or one provided by Cypress via Docker Hub. These Docker images, not only furnished by Cypress,
+include various versions of Chrome, Firefox, and Edge.
+
+Executing the following command initiates a headless run against the Electron browser. Upon completion, a succinct
+table is generated, detailing the number of tests for each 'cy.js' file, in addition to the failed, skipped and passed
+tests with the run time of each specific 'cy.js' file. At the bottom of the table, a comprehensive summary displays the
+total number of tests run, the duration of the run, and the counts of failed, passed, and skipped tests.
+
+```bash
+npx cypress run
+```
+
+This would completely replace the Selenium step in your current CI/CD workflow.
+
+Learn more about running Cypress tests in Continuous Integration [here](/guides/continuous-integration/introduction).
```

Comment:
GTM can chime in here but I believe this allows for better SEO when crawling these pages.

```suggestion
Learn more about [running Cypress tests in Continuous Integration](/guides/continuous-integration/introduction).
```